.travis.yml

```diff
@@ -6,7 +6,7 @@ services:
 language: go
 
 go:
-  - 1.13.5
```

Comment:
since the required go version of k8s is 1.14.4, we should update it in travis too.